express_relay/sdk/python/searcher/examples/simple_searcher.py

```diff
@@ -0,0 +1,160 @@
+        Individual searchers will have their own methods to determine market impact and the profitability of conducting a liquidation. This function can be expanded to include external prices to perform this evaluation.
+        In this simple searcher, the function always (naively) returns a BidInfo object with the default bid and a valid_until timestamp.
+        Args:
+            default_bid: The default amount of bid for liquidation opportunities.
+            opp: A OpportunityParamsWithMetadata object, representing a single liquidation opportunity.
+        Returns:
+            If the opportunity is deemed worthwhile, this function can return a BidInfo object, whose contents can be submitted to the auction server. If the opportunity is not deemed worthwhile, this function can return None.
+        """
+        signature_liquidator = self.construct_signature_liquidator(
+            opp,
+            self.default_bid,
+            VALID_UNTIL,
+        )
+
+        opportunity_bid = {
+            "permission_key": opp.permission_key,
+            "amount": str(self.default_bid),
+            "valid_until": str(VALID_UNTIL),
+            "liquidator": self.liquidator,
+            "signature": bytes(signature_liquidator.signature).hex(),
+        }
+        opportunity_bid = OpportunityBid.from_dict(opportunity_bid)
+
+        bid_info = BidInfo(
+            opportunity_id=opp.opportunity_id,
+            opportunity_bid=opportunity_bid,
+        )
+
+        return bid_info
+
+    async def ws_opportunity_handler(
+        self, opp: OpportunityParamsWithMetadata
+    ):
+        bid_info = self.assess_liquidation_opportunity(opp)
+        resp = await self.submit_bid(bid_info)
+        logger.info(
+            "Submitted bid amount %s for opportunity %s, server response: %s",
+            bid_info.opportunity_bid.amount,
+            bid_info.opportunity_id,
+            resp.text,
+        )
+
+
+async def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--verbose", action="count", default=0)
+    parser.add_argument(
+        "--private-key",
+        type=str,
+        required=True,
+        help="Private key of the searcher for signing calldata",
+    )
+    parser.add_argument(
+        "--chain-id",
+        type=str,
+        required=True,
+        help="Chain ID of the network to monitor for liquidation opportunities",
+    )
+    parser.add_argument(
+        "--bid",
+        type=int,
+        default=10,
+        help="Default amount of bid for liquidation opportunities",
+    )
+    parser.add_argument(
+        "--liquidation-server-url",
+        type=str,
+        required=True,
+        help="Liquidation server endpoint to use for fetching opportunities and submitting bids",
+    )
+    parser.add_argument(
+        "--use-ws",
```

Comment:
we should encourage users to do things the right way. and the right way here it to just use websockets instead of polling. So let's remove this argument (with lines 123-156) and always use websocket.